<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QDialog" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>430</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Deposit Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/rsc/Icons/calc_icon.png</normaloff>:/rsc/Icons/calc_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(226, 226, 226);

</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>304</width>
     <height>320</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>304</width>
     <height>320</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 14px;
	border: 2px solid gray;
	border-color: #d09d80;
	border-radius: 15px;
	margin-top: 10px;
}

QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top center;
	padding: 0px 5px;
}
</string>
   </property>
   <property name="title">
    <string>Расчётные данные</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="text">
         <string>Начисленные проценты (руб.):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="output_percentsum">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: rgb(226, 104, 126);
	background-color: rgb(255, 255, 255);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>60</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="text">
         <string>Сумма налога (руб.):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="output_tax">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: rgb(226, 104, 126);
	background-color: rgb(255, 255, 255);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>60</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Итого на конец срока (руб.):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="output_total">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: rgb(226, 104, 126);
	background-color: rgb(255, 255, 255);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>305</width>
     <height>360</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>304</width>
     <height>360</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 14px;
	border: 2px solid gray;
	border-color: #d09d80;
	border-radius: 15px;
	margin-top: 10px;
}

QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top center;
	padding: 0px 5px;
}
</string>
   </property>
   <property name="title">
    <string>Введите исходные параметры</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>145</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>145</width>
          <height>16</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="text">
         <string>Сумма вклада (руб.):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_sum">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: rgb(93, 87, 104);
	background-color: rgb(255, 255, 255);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="locale">
         <locale language="C" country="AnyTerritory"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>145</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>145</width>
          <height>16</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="text">
         <string>Срок размещения (мес.):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_period">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: rgb(93, 87, 104);
	background-color: rgb(255, 255, 255);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="locale">
         <locale language="C" country="AnyTerritory"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>145</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>145</width>
          <height>16</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="text">
         <string>Ставка процента (%):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_percent">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: rgb(93, 87, 104);
	background-color: rgb(255, 255, 255);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="locale">
         <locale language="C" country="AnyTerritory"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>145</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>145</width>
          <height>16</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="text">
         <string>Налоговая ставка (%):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_tax_rate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: rgb(93, 87, 104);
	background-color: rgb(255, 255, 255);
	font-weight: bold;
	font-size: 12px;
}</string>
        </property>
        <property name="locale">
         <locale language="C" country="AnyTerritory"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="check_put">
      <property name="styleSheet">
       <string notr="true">font-weight: bold;
font-size: 12px;
color: rgb(93, 87, 104);</string>
      </property>
      <property name="text">
       <string>Возможность пополнения</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="put_sum">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>22</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	color: rgb(93, 87, 104);
	background-color: rgb(255, 255, 255);
	font-weight: bold;
	font-size: 12px;
}</string>
      </property>
      <property name="locale">
       <locale language="C" country="AnyTerritory"/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_draw">
      <property name="styleSheet">
       <string notr="true">font-weight: bold;
font-size: 12px;
color: rgb(93, 87, 104);</string>
      </property>
      <property name="text">
       <string>Возможность частичного снятия</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="draw_sum">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>22</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	color: rgb(93, 87, 104);
	background-color: rgb(255, 255, 255);
	font-weight: bold;
	font-size: 12px;
}</string>
      </property>
      <property name="locale">
       <locale language="C" country="AnyTerritory"/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
	color: rgb(93, 87, 104);
	font-weight: bold;
	font-size: 12px;
	border: 1px solid gray;
	border-radius: 0px
}

QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top left;
}
</string>
      </property>
      <property name="title">
       <string>Капитализация процентов</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QRadioButton" name="capital_month">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
font-size: 12px;
color: rgb(93, 87, 104);</string>
         </property>
         <property name="text">
          <string>Ежемесячная</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="capital_quarter">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
font-size: 12px;
color: rgb(93, 87, 104);</string>
         </property>
         <property name="text">
          <string>Ежеквартальная</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="capital_none">
         <property name="styleSheet">
          <string notr="true">font-weight: bold;
font-size: 12px;
color: rgb(93, 87, 104);</string>
         </property>
         <property name="text">
          <string>Нет</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_exit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	text-align: center;
	font-weight: bold;
	color: rgb(255, 255, 255);
	background-color: rgb(93, 87, 104);
	border-radius: 8px;
}
QPushButton:hover {
	background-color: rgb(143, 143, 143);
}
QPushButton:pressed {
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_calc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	text-align: center;
	font-weight: bold;
	color: rgb(255, 255, 255);
	background-color: rgb(93, 87, 104);
	border-radius: 8px;
}
QPushButton:hover {
	background-color: rgb(143, 143, 143);
}
QPushButton:pressed {
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Расчёт</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>300</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: rgb(60, 60, 60);
	font-weight: bold;
	border-color: 2px, rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:700; font-style:italic;&quot;&gt;* Сумма пополнения/снятия из расчёта:&lt;br /&gt;- 1 раз в месяц &lt;br /&gt;(при ежемясячной капитализации процетов),&lt;br /&gt;- 1 раз в квартал &lt;br /&gt;(при ежеквартальной капиталиации процентов).&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
